packages/website/docs/adding-recent-searches.md

```diff
@@ -68,6 +68,51 @@ Since the `recentSearchesPlugin` reads from [`localStorage`](https://developer.m
   openOnFocus={true}
 />
 
+## Transforming recent searches
```

Comment:
Readers might not realize why they would want to "transform" recent searches, but they could want to customize them.

```suggestion
## Customizing recent searches
```